### ISSUE TITLE
Support slice of files and includes

### ISSUE THREAD

**Issue by emoon** [alexcrichton/cc-rs#352]
Something that would be nice would be to allow `includes(incs: &[PathBuf])` and something similar for `files` instead of just one include dir and one file at a time.

While it's not that hard to write

```Rust
let some_include_dirs = [
  "dir0", "dir1", "dir2"
];

for dir in &some_include_dirs {
    build.include(dir);
}
```

If the user could supply an array/slice directly it would reduce the code a bit.

**Comment by alexcrichton** [comment]
Sounds reasonable to me to support this!

**Comment by dtolnay** [comment]
`files` was implemented in #205. I've opened #550 for `includes` after having looked for this myself.
